i18n/locales/ja/ch-demos/piday-code.ipynb

```diff
@@ -152,10 +152,12 @@
    "source": [
     "## 実機かシミュレーターでQiskitジョブの実行\n",
     "\n",
-    "def run_job(circ_, backend_, shots_=1000, optimization_level_=0):\n",
-    "    job = execute(circ_, backend=backend_, shots=shots_, optimization_level=optimization_level_)\n",
+    "def run_job(circ, backend, shots=1000, optimization_level=0):\n",
```

Comment:
@frankharkins , According to this change, line 230 should be changed to `counts = run_job(circ, backend=simulator, shots=10000, optimization_level=0)` from `counts = run_job(circ, backend_=simulator, shots_=10000, optimization_level_=0)`.